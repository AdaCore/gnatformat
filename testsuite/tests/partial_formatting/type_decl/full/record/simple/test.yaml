description: ConcreteTypeDecl formatting for line selection.
driver: partial_gnatformat
args:
  [
    "test.ads",
    "-SL 61",
    "-EL 63",
  ]
status_code: 1