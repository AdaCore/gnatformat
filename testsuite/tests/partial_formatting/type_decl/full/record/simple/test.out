<CompilationUnit test.ads:27:1-149:23>
test.ads(<ConcreteTypeDecl ["Cell"] test.ads:61:4-65:15>) - 61:1-65:15
^
   type Cell is
     record
       Value : Integer;
       Succ  : Link;
       Pred  : Link;
     end record;$

